libs/server-sdk/include/launchdarkly/server_side/integrations/iserialized_item_kind.hpp

```diff
@@ -0,0 +1,43 @@
+#pragma once
```

Comment:
This lives in the public `include` directory because it will need to be consumed by implementers of persistent data stores.